### ISSUE TITLE
any installation description available?

### ISSUE THREAD

**Issue by feiercrack** [DrParanoia/ed-rgb#3]
how can i install this on my Win10 x64 1903 for the Logitech G910?

**Comment by TomW1605** [comment]
there is a .jar file [here](https://github.com/DrParanoia/ed-rgb/blob/tom_develop/ed-logitech-rgb.jar?raw=true). This is for my development branch. there are a few things that are difrent to the main b ut mostly they are athetic (key colours and the like) or additional features. to my knolage this .jar is stable. if u have any trouble or want one for the main branch the let me know.

i am closing this issue but reopen if nesasery

**Comment by feiercrack** [comment]
got it installed. but i use an different keyboard binding file from aussiedroid to work with my Warthog HOTAS.

if the java window is primary, it works. when i swap over to main game window, it stopped...
any tip?
[screenoutput.txt](https://github.com/DrParanoia/ed-rgb/files/3487192/screenoutput.txt)

**Comment by TomW1605** [comment]
tl;dr redownload the .jar and launch it with the command line option -alwayshighlight

the program is designed to only highlight when ether itself or the game is active however in this case it appiers that when you switch to the game it is not relising it. i have had a look at aussiedroid profile u r using and i think the profile isent the problem but the script that goes with it might be causing an issue. not sure how it works but if it is stealing the keybord focus away from the game then that would make the program think the game is not focused. i have added a command line flag that dissables this behaviour.

**Comment by feiercrack** [comment]
works now!
thanxx

**Comment by TomW1605** [comment]
great. if you have any issues, segestions or general feedback please make an issue here or message me on discord `@​Tom#7765`. as there are curently only about 3-4 people who use this it is hard to improve or fix issues so any feedback would be appreciated.
